pjsip/src/pjsip/sip_msg.c

```diff
@@ -885,6 +885,38 @@ static pjsip_generic_int_hdr* pjsip_generic_int_hdr_shallow_clone( pj_pool_t *po
     return hdr;
 }
 
+
+///////////////////////////////////////////////////////////////////////////////
+/*
+ * Print function for a generic array header with an arbitrary delimiter.
+ * The generic array header will use this print function with ', ' as a delimiter,
+ * the privacy header will use it with '; '.
+ */
+int pjsip_generic_array_hdr_delimited_print( pjsip_generic_array_hdr *hdr,
+                      char *buf, pj_size_t size, char *delimiter, size_t delimiter_length)
+{
+    char *p = buf, *endbuf = buf+size;
```

Comment:
This was terrible before you moved it. If you fancy fixing it up:

- Avoid multiple declarations when one of them is a pointer - use two lines instead.
- You want spaces around the plus symbol.